conans/test/functional/configuration/requester_test.py

```diff
@@ -0,0 +1,82 @@
+# coding=utf-8
+
+import os
+import unittest
+
+import six
+from mock import Mock
+
+from conans.client.conan_api import Conan
+from conans.client.tools import environment_append
+from conans.client.tools.files import replace_in_file, save
+from conans.errors import ConanException
+from conans.paths import CACERT_FILE
+from conans.test.utils.tools import TestBufferConanOutput, temp_folder
+
+
+class MockRequesterGet(Mock):
+    verify = None
+
+    def get(self, url, **kwargs):
+        self.verify = kwargs.get('verify', None)
+
+
+class ConanRequesterCacertPathTests(unittest.TestCase):
```

Comment:
These tests are clearly insufficient. The feature is broken for an unexisting cert file. All is needed is to test it manually.
Reasons:
- The exception of not existing file is raised when ConanRequester is created
- ConanRequester is created first thing by the ConanAPI
- If something fails there, it is captured and silented and a MIGRATION_ERROR return code is returned

Thus, commands fail completely silently for users with cacert_path=something/does/not/exist.

This could have been detected by an integration test. New features always require integrations tests.